packages/website/docs/who-can-apply.md

```diff
@@ -1,61 +1,41 @@
 ---
 title: Who can apply?
+authors:
+  - name: Clément Vannicatte
+    title: Software Engineer @ Algolia
+    url: https://github.com/shortcuts
 ---
 
-We built DocSearch from the ground up with the idea of improving search on large
-technical documentations. For this reason, we are offering a free hosting
-version to all online technical documentations.
+We built DocSearch from the ground up with the idea of improving search on large technical documentations. For this reason, we are offering a free hosting version to all online technical documentations of open-source projects and technical blogs.
 
-We're always sad to have to turn down applications, but with the number of
-requests we receive every day, we've had to focus on fair criteria. We hope you
-understand.
+We're always sad to have to turn down applications, but with the number of requests we receive every day, we've had to focus on fair criteria. We hope you understand.
 
 ## The checklist
 
-To have your request validated, we'll ask that you check all the following
-points.
+To have your request validated, we'll ask that you check all the following points.
 
-- You must be the **owner** of the website, or at least have the permissions to
-  update its content. You'll have to [include a JavaScript snippet][1] to enable
-  DocSearch.
+- You must be the **owner** of the website, or at least have the permissions to update its content. You'll have to [include a JavaScript snippet][1] to enable DocSearch.
```

Comment:
Not a good suggestion, but it currently seems it's mandatory while it's not
```suggestion
- You must be the **owner** of the website, or at least have the permissions to update its content. You'll need to [include a JavaScript snippet][1] to power your DocSearch frontend.
```

Reply:
```suggestion
- You must be the **owner** of the website, or at least have the permissions to update its content. You'll need to [include a JavaScript snippet][1] to your frontend to implement DocSearch.
```